app/src/main/res/values/dimens.xml

```diff
@@ -420,6 +420,9 @@
   <dimen name="faq_item_header_margin_end">24dp</dimen>
   <dimen name="faq_item_header_margin_bottom">8dp</dimen>
 
+  <!-- Help -->
```

Comment:
Rename to `<!-- NavigationDrawer -->` considering this same variable will get used at multiple places

Reply:
Done